src/lib.rs

```diff
@@ -1839,6 +1840,37 @@ impl OpAttrDef {
     }
 }
 
+/// Types of attribute values for an operation
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum AttrValue {
+    S(Vec<u8>), 
+    I(i64), 
+    F(f32), 
+    B(bool), 
+    Field_type(protos::types::DataType), 
+    Shape(Shape), 
+    Tensor(protos::tensor::TensorProto), 
+    List(AttrValue_ListValue), 
+    Func(NameAttrList), 
+    Placeholder(String),
+}
+
+#[derive(Debug)]
+pub struct NameAttrList {
+    // message fields
+    pub name: String,
+    // TODO: the AttrValue type in this hashmap is the incorrect type. Please ref to
+    // protos/attr_value.rs:1381
+    pub attr: HashMap<::std::string::String, AttrValue>,
+    // special fields
+    pub unknown_fields: UnknownFields,
+    pub cached_size: CachedSize,
+}
+
+#[derive(Debug)]
+pub struct AttrValue_ListValue {}
```

Comment:
This doesn't need to use exactly the same name as the proto, especially since the generated names are a weird mix of CamelCase and snake_case.  This likely makes more sense as either `AttrValueList` or `AttrListValue`.